firrtl/src/main/scala/firrtl/ir/Serializer.scala

```diff
@@ -280,7 +280,8 @@ object Serializer {
       writers.foreach { w => b ++= "writer => "; b ++= legalize(w); newLineAndIndent(1) }
       readwriters.foreach { r => b ++= "readwriter => "; b ++= legalize(r); newLineAndIndent(1) }
       b ++= "read-under-write => "; b ++= readUnderWrite.toString
-    case Attach(info, exprs) =>
+    case DefTypeAlias(info, name, tpe) => b ++= "type "; b ++= name; b ++= " = "; s(tpe); s(info)
```

Comment:
```suggestion
    case DefTypeAlias(info, name, tpe) => b ++= "type "; b ++= name; b ++= " = "; s(tpe);
```

Info locators are not supported on type alias statements.  They're not in the spec grammar and firtool rejects them, so probably best to omit.

(not sure whether this removal should propagate up to remove it from `DefTypeAlias` and etc., but maybe?)